harvey/setrun.py

```diff
@@ -524,8 +541,8 @@ def setgeo(rundata):
                                   [np.infty, 0.0, -np.infty],
                                   [0.030, 0.022]])
 
-    # Texas gulf coast
-    data.friction_regions.append([(-99.2, 26.4), (-94.2, 30.4),
+    # La-Tex Shelf
+    data.friction_regions.append([(-98, 25.25), (-90, 30),
```

Comment:
This really should not be changed to match the La-Tex shelf that you are plotting but an actual region where the shelf is "slipperier".